src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -78,6 +78,8 @@
     )
 
     var (
+        Done = errors.New("iterator done")
```

Comment:
Hm, I've missed the design doc, sorry -- inviting @jba to make sure.

Also --
- you should put comment above the definition, so godoc can show what this constant means.
- I prefer not to define this if the service does not have page-streaming methods (so far all services have page-streaming, but there can be)